### ISSUE TITLE
Code breaking with webpack 2

### ISSUE THREAD

**Issue by atkawa7** [tannerlinsley/react-table#28]
I have upgrade my dependencies. It seems your code breaks around this [line](http://github.com/tannerlinsley/react-table/blob/master/src/index.js#L674). 

```
let visibleColumns = decoratedColumns.slice()
    let allVisibleColumns = []

    visibleColumns = visibleColumns.map((column, i) => {
      if (column.columns) {
        const visibleSubColumns = column.columns.filter(d => pivotBy.indexOf(d.id) > -1 ? false : _.getFirstDefined(d.show, true))
        return {
          ...column,
          columns: visibleSubColumns
        }
      }
      return column
    })
```
![code](https://cloud.githubusercontent.com/assets/5673828/22049857/8263f9e4-dcf3-11e6-8c34-37f18ed845f8.png)

**Comment by tannerlinsley** [comment]
Can you post your props and data?

**Comment by atkawa7** [comment]
here is the function that creates columns
```
function getColumns(route) {
const columns = [  'date',
    'total_plays',
    'total_views',
    'total_time',
    'title',
    'plays'];
    var headers = columns.map(function(header) {
        if (header === 'total_time') {
            return {
                header: titleCaps(header),
                accessor: header,
                render: props => (
                    <span>
                        {convertHours(parseInt(props.value))}
                    </span>
                )
            };
        } else if (header === 'title') {
            return {header: '#1 Title ', accessor: header}
        } else if (header === 'plays') {
            return {header: '#1 Plays ', accessor: header}
        } else if (header === 'date') {
            return {
                header: titleCaps(header),
                accessor: header,
                render: props => (
                    <span>
                        <Link to={`${route.path}/${props.value}`}>{props.value}</Link>
                    </span>
                )
            };
        } else if (header === 'title') {

            return {
                header: titleCaps(header),
                accessor: header,
                render: props => <span className='number'>{props.value.split(/\s+/).slice(0, 10).join(' ')}</span>,
                sortable: false
            }
        } else {

            return {header: titleCaps(header), accessor: header};
        }

    });
    return headers;
    console.log(headers);
}```

**Comment by tannerlinsley** [comment]
Looks good to me.  What about where you render your table?  Maybe set a break point at the line and log some of the variables?

**Comment by atkawa7** [comment]
Here is the chart without the table but once I add table it does not work

![screenshot from 2017-01-17 20-54-32](https://cloud.githubusercontent.com/assets/5673828/22050325/36e01972-dcf7-11e6-8a93-4d8c30abb24a.png)

**Comment by atkawa7** [comment]
<ReactTable columns={currentColumns} manual data={this.state.results} pages={this.state.pageNumber} loading={this.state.loading} onChange={this.handlePage.bind(this)} minRows={0} pageSizeOptions={[
                    5,
                    10,
                    20,
                    25,
                    50,
                    100,
                    150,
                    200,
                    250,
                    500
               ]}/>

**Comment by atkawa7** [comment]
In the default constructor 
```
this.state ={
  results: [],
    pageNumber: 1,
    page: 1,
    count: 0,
    next: null,
    previous: null,
    pageSize: 20,
    loading: true,
    ordering: '-date'};
```

**Comment by tannerlinsley** [comment]
Set a breakpoint at the error and copy the current internal state and props of the table.  Post them here if you can?

**Comment by atkawa7** [comment]
Found the error. It is because of this import function. It is transforming the code

![screenshot from 2017-01-17 21-26-42](https://cloud.githubusercontent.com/assets/5673828/22050856/bf0b4bec-dcfb-11e6-9fed-ca88b802cb92.png)

**Comment by tannerlinsley** [comment]
Well that's weird.  Very standard import, no?

**Comment by dmftuva** [comment]
export default class SettingsOrgs extends React.Component {

    drawTable() {
        var columns = [
            {
                header: 'ID',
                accessor: 'id'
            },
            {
                header: 'Full Name',
                accessor: 'fullname'
            },
            {
                header: 'Inn',
                accessor: 'inn'
            },
        ];

        return (
            <div>
                <ReactTable
                    data={this.state.items}
                    columns={columns}
                />
            </div>
        )
    }

    render() {
        return (
               <div className='row expanded'>
                    <div className="small-12">
                        {this.drawTable()}
                    </div>
                </div>
   )
  }
}

Uncaught (in promise) TypeError: Cannot read property 'columns' of undefined

**Comment by dmftuva** [comment]
import React,{Component} from 'react'
import ReactTable from 'react-table'

**Comment by vladra** [comment]
I've just encountered same error, with webpack 1. So it's not only webpack 2 related

**Comment by atkawa7** [comment]
What is column property?

**Comment by tannerlinsley** [comment]
The storybook is working fine with examples similar to yours.  Does the Server-side demo work for you?

**Comment by tannerlinsley** [comment]
Wait.  I think I see the problem.  I'll have the fix out in 5 minutes.

**Comment by tannerlinsley** [comment]
Fixed in 505db8d43ed0b545d48f0ef2b752c2877368fcca (react-table@4.2.4)

**Comment by tannerlinsley** [comment]
Thanks for finding this bug!
